packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCard.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <div :class="$style['c-restaurantCard']">
+        <image-block :class="$style['c-skeletonLoader-background']" />
+        <restaurant-card-header :class="$style['c-skeletonLoader-header']" />
+        <restaurant-card-data :class="$style['c-skeletonLoader-data']" />
+    </div>
```

Comment:
Sam here – as the child classes are all inside `c-restaurantCard`, can just name these related to that.

```suggestion
    <div :class="$style['c-restaurantCard']">
        <image-block :class="$style['c-restaurantCard-background']" />
        <restaurant-card-header :class="$style['c-restaurantCard-header']" />
        <restaurant-card-data :class="$style['c-restaurantCard-data']" />
    </div>
```

Reply:
Thanks @ashleynolan that makes much more sense :)